TaskSharper.Calender.WPF/ViewModels/TodayViewModels/CalendarDateViewModel.cs

```diff
@@ -8,5 +9,32 @@ public class CalendarDateViewModel : BindableBase
 
         public string DayOfWeek { get; set; } = DateTime.Now.DayOfWeek.ToString();
         public int DayOfMonth { get; set; } = DateTime.Now.Day;
+
+        public CalendarDateViewModel()
+        {
+            
+        }
+
+        public CalendarDateViewModel(int dayOfMonth = 0)
+        {
+            SetDate(dayOfMonth);
+        }
+
+        private void SetDate(int today)
+        {
+            var dayOffset = today - (int)DateTime.Now.DayOfWeek;
+
+            // TODO:: Dont do this.. do something smarter
```

Comment:
Last week mads mentioned as smart solution to this. 
Has this been implemented? If yes, remove comment.